web/src/components/pages/profile/avatar-setup/avatar-setup.tsx

```diff
@@ -145,6 +259,25 @@ class AvatarSetup extends React.Component<Props> {
           </label>
         </div>
 
+        <div style={{ display: isProduction() ? 'none' : 'block' }}>
```

Comment:
Another way to do this is to write:
```jsx
{isProduction() && <div>someContent</div>}
```
Which decreases the amount of things that need to be rendered and computed in the false-branch of that conditional. But since this is non-critical here, I'll just merge it in.

Reply:
Got it ! double &&